app/src/main/java/ch/epfl/sweng/SDP/home/HomeActivity.java

```diff
@@ -189,7 +189,7 @@ private void listenerEventSelector(final View view, int id) {
                 profileWindow.dismiss();
                 break;
             case R.id.practiceButton:
-                launchActivity(DrawingOffline.class);
+                launchActivity(DrawingOfflineItems.class);
```

Comment:
We should create a separate button for this mode

Reply:
Yeah I’ll try to design a new button whenever I have time